python/cudf/cudf/core/dataframe.py

```diff
@@ -1694,7 +1694,16 @@ def _clean_nulls_from_dataframe(self, df):
                 # TODO we need to handle this
                 pass
             elif df._data[col].has_nulls():
-                df[col] = df._data[col].astype("str").fillna(cudf._NA_REP)
+                if isinstance(
+                    df._data[col],
+                    (
+                        cudf.core.column.DatetimeColumn,
+                        cudf.core.column.TimeDeltaColumn,
+                    ),
+                ):
+                    df[col] = df._data[col].astype("str").fillna(cudf._NAT_REP)
+                else:
+                    df[col] = df._data[col].astype("str").fillna(cudf._NA_REP)
```

Comment:
Could we write this like
```python
fill_value = cudf._NAT_REP if ... else cudf._NA_REP
df[col] = df._data[col].astype("str").fillna(fill_value)
```
Just want to avoid repeating the bulk of the `fillna` logic. Only the value changes.

Reply:
Or even copy the style in `core/index.py` from this PR:
```python
df[col] = df._data[col].astype("str").fillna(cudf._NAT_REP if ... else cudf._NA_REP)
```

Reply:
Done 👍